### PR TITLE
Bug 1873372 - [RHOCP4.5] Fix "Exposing the log store service as a rou…

### PR DESCRIPTION
**Pull request by rolfedh**
…te" doc of OCP4.5 as to use infra-NNN

- For versions 4.5+
- https://bugzilla.redhat.com/show_bug.cgi?id=1873372
- Direct link to doc preview: https://deploy-preview-30498--osdocs.netlify.app/openshift-enterprise/latest/logging/config/cluster-logging-log-store.html#cluster-logging-elasticsearch-exposing_cluster-logging-store
- Ready for peer review and merge.

### CODE REVIEW

**Comment by rolfedh** [comment]
Hi, @periklis. I've marked a handful of instances that look like they might need to be updated from `.operator` to `infra-`. Would you mind reviewing these and making recommendations?

**Comment by periklis** [comment]
@rolfedh FWIW, if the change is only the one curl call then looks good to me. However, I don't have access to the BZ https://bugzilla.redhat.com/show_bug.cgi?id=1873372 to conclude that everything is solved.

File: modules/cluster-logging-updating-logging.adoc [hunk 1]
```diff
@@ -148,6 +148,7 @@ green  open   infra-000008
 green  open   infra-000004                                                          rtDSzoqsSl6saisSK7Au1Q   3 1       226717            0        297            148
 green  open   infra-000012                                                          RSf_kUwDSR2xEuKRZMPqZQ   3 1       227623            0        295            147
 green  open   .kibana_7                                                             1SJdCqlZTPWlIAaOUd78yg   1 1            4            0          0              0
+// TBD - Update from `.operations` to `infra-` in the following example for https://bugzilla.redhat.com/show_bug.cgi?id=1873372
 green  open   .operations.2020.06.30                                                aOHMYOa3S_69NJFh2t3yrQ   3 1      4206118            0       8998           4499
```
Comment (periklis) [inline modules/cluster-logging-updating-logging.adoc:152]: You can delete from the sample output lines that include `.project...` and `.operations...`. AFAICS, we already have `.infra-0000..` (corresponds to old `.operations...`) and `.app-0000*` (corresponds to old `.project...`).

File: modules/cluster-logging-elasticsearch-exposing.adoc [hunk 2]
```diff
@@ -177,6 +177,7 @@ The response appears similar to the following:
     "max_score": 1,
     "hits": [
       {
+        // TBD - Update from `.operations` to `infra-` in the following example for https://bugzilla.redhat.com/show_bug.cgi?id=1873372
```
Comment (periklis) [inline modules/cluster-logging-elasticsearch-exposing.adoc:180]: You can use the up2date curl from this PR above to provide a fresh example output.

File: modules/cluster-logging-curator-delete-index.adoc [hunk 3]
```diff
@@ -47,6 +47,7 @@ The available parameters are:
 |`project_name`
 |The actual name of a project, such as *myapp-devel*. For {product-title} *operations*
 logs, use the name `.operations` as the project name.
+// TBD - Update from `.operations` to `infra` for https://bugzilla.redhat.com/show_bug.cgi?id=1873372?
```
Comment (periklis) [inline modules/cluster-logging-curator-delete-index.adoc:50]: I am not sure if we can really drop or change in the curator configuration here. It looks dedicated to curate the old pre-4.5 indices `.project...` and `.operations...`. I would pass this to @jcantrill to answer.

File: modules/cluster-logging-elasticsearch-exposing.adoc (hunk 2 above)
Comment (rolfedh) [reply modules/cluster-logging-elasticsearch-exposing.adoc:180]: Sorry - I'm working to get a realistic environment set up that can generate such an output. However, I'm not there yet. I'll create an approximation of the output for now.

File: modules/cluster-logging-curator-delete-index.adoc (hunk 3 above)
Comment (rolfedh) [reply modules/cluster-logging-curator-delete-index.adoc:50]: Good catch. I've removed that comment and confirmed this example is for pre-4.5 indices.

**Comment by rolfedh** [comment]
Hi, @anpingli. Would you review and approve this PR?

File: modules/cluster-logging-elasticsearch-exposing.adoc [hunk 4]
```diff
@@ -152,7 +152,7 @@ $ routeES=`oc get route elasticsearch -o jsonpath={.spec.host}`
 +
 [source,terminal]
 ----
-$ curl -tlsv1.2 --insecure -H "Authorization: Bearer ${token}" "https://${routeES}/.operations.*/_search?size=1" | jq
+$ curl -tlsv1.2 --insecure -H "Authorization: Bearer ${token}" "https://${routeES}/infra-*/_search?size=1" | jq
```
Comment (anpingli) [inline modules/cluster-logging-elasticsearch-exposing.adoc:155]: How about use the command below?  use jq is not a good idea.
curl -tlsv1.2 --insecure -H "Authorization: Bearer ${token}" "https://${routeES}"
{
  "name" : "elasticsearch-cdm-i40ktba0-1",
  "cluster_name" : "elasticsearch",
  "cluster_uuid" : "0eY-tJzcR3KOdpgeMJo-MQ",
  "version" : {
    "number" : "6.8.1",
    "build_flavor" : "oss",
    "build_type" : "zip",
    "build_hash" : "Unknown",
    "build_date" : "Unknown",
    "build_snapshot" : true,
    "lucene_version" : "7.7.0",
    "minimum_wire_compatibility_version" : "5.6.0",
    "minimum_index_compatibility_version" : "5.0.0"
  },
  "tagline" : "You Know, for Search"
}

File: modules/cluster-logging-updating-logging.adoc [hunk 5]
```diff
@@ -148,8 +148,6 @@ green  open   infra-000008
 green  open   infra-000004                                                          rtDSzoqsSl6saisSK7Au1Q   3 1       226717            0        297            148
 green  open   infra-000012                                                          RSf_kUwDSR2xEuKRZMPqZQ   3 1       227623            0        295            147
 green  open   .kibana_7                                                             1SJdCqlZTPWlIAaOUd78yg   1 1            4            0          0              0
-green  open   .operations.2020.06.30                                                aOHMYOa3S_69NJFh2t3yrQ   3 1      4206118            0       8998           4499
-green  open   project.local-storage.d5c8a3d6-30a3-4512-96df-67c537540072.2020.06.30 O_Uldg2wS5K_L6FyqWxOZg   3 1        91052            0        135             67
 green  open   infra-000010                                                          iXwL3bnqTuGEABbUDa6OVw   3 1       248368            0        317            158
 green  open   .searchguard                                                          rQhAbWuLQ9iuTsZeHi_2ew   1 1            5           64          0              0
```
Comment (anpingli) [inline modules/cluster-logging-updating-logging.adoc:152]: Please delete .searchguard   too.

File: modules/cluster-logging-updating-logging.adoc [hunk 6]
```diff
@@ -159,19 +157,16 @@ green  open   .orphaned.2020.06.30
 green  open   infra-000007                                                          llkkAVSzSOmosWTSAJM_hg   3 1       228584            0        296            148
 green  open   infra-000005                                                          d9BoGQdiQASsS3BBFm2iRA   3 1       227987            0        297            148
 green  open   .kibana.647a750f1787408bf50088234ec0edd5a6a9b2ac                      l911Z8dSI23py6GDtyJrA    1 1            5            4          0              0
```
Comment (anpingli) [inline modules/cluster-logging-updating-logging.adoc:159]: Delete  .kibana.647a750f1787408bf50088234ec0edd5a6a9b2ac

File: modules/cluster-logging-elasticsearch-exposing.adoc (hunk 4 above)
Comment (rolfedh) [reply modules/cluster-logging-elasticsearch-exposing.adoc:155]: Excellent - thanks.

File: modules/cluster-logging-updating-logging.adoc (hunk 5 above)
Comment (rolfedh) [reply modules/cluster-logging-updating-logging.adoc:152]: Will do. Thanks.

File: modules/cluster-logging-updating-logging.adoc (hunk 6 above)
Comment (rolfedh) [reply modules/cluster-logging-updating-logging.adoc:159]: Will do. Thanks.

File: modules/cluster-logging-elasticsearch-exposing.adoc (hunk 4 above)
Comment (rolfedh) [reply modules/cluster-logging-elasticsearch-exposing.adoc:155]: @anpingli  Sorry. The formatting is off. Would you resend this using markdown code block (between triple-backticks, like this ``` )?

Comment (rolfedh) [reply modules/cluster-logging-elasticsearch-exposing.adoc:155]: @anpingli Please have a look at the updated command and example output. To get the correct formatting, would it be possible to provide the example output between triple backticks (```) like this?
``` 
backticks 
```

**Comment by rolfedh** [comment]
@anpingli  Please review and approve with `/lgtm`. Thank you.

**Review by anpingli — COMMENTED** [review]
/lgtm

**Comment by bmcelvee** [comment]
LGTM!

**Pull request merged**